tests/generic_config_updater/test_pfcwd_status.py

```diff
@@ -105,7 +105,8 @@ def extract_pfcwd_config(duthost, start_pfcwd):
                       'restore_time' as the 2nd level keys
     """
     output = duthost.command('show pfcwd config')
-    pytest_assert('Ethernet' in output['stdout'], 'No ports found in the pfcwd config')
+    if 'Ethernet' not in output['stdout']:
+        pytest.skip('No ports found in the PFC_WD config, skipping GCU PFC_WD test')
```

Comment:
>skip [](http://example.com/codeflow?start=15&length=4)

Instead of skipping, could we test the result is command return failure, and treat it as expected?